<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>54</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>54</height>
   </size>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>170</red>
        <green>170</green>
        <blue>170</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>127</red>
        <green>127</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <item>
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>381</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="sizeIncrement" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>TOTAL</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fee_lineEdit" >
     <property name="minimumSize" >
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="ptfee_lineEdit" >
     <property name="minimumSize" >
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>120</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
